pkg/pod/status.go

```diff
@@ -208,7 +208,7 @@ func setTaskRunStatusBasedOnStepStatus(ctx context.Context, logger *zap.SugaredL
 				}
 
 				taskResults, filteredResults := filterResults(results, specResults)
-				if tr.IsSuccessful() {
+				if tr.IsDone() {
```

Comment:
I think you need to do the same thing in line 185 as well.

Reply:
That will account for the same change when fetching results from sidecar logs too.

Reply:
thanks a bunch @chitrangpatel for a quick response 🙏  hopefully works now 🤞